tests/lib.rs

```diff
@@ -137,11 +137,17 @@ fn cursor_workflow() {
 #[test]
 fn statement_columns() {
     let connection = setup_users(":memory:");
-    let statement = "SELECT * FROM users";
+    let statement = "SELECT id, name, age, photo as user_photo FROM users";
     let mut statement = ok!(connection.prepare(statement));
 
     assert_eq!(statement.columns(), 4);
 
+    {
```

Comment:
Why is it in a separate block?

Reply:
But in principle, `statement_columns` is supposed to test `columns`. I think the testing of `column_name` should live in its own method, `statement_column_name`.

Reply:
it was because column_names would borrow self, probably because column_name returns a &str bound to the statement. i moved to a new test anyway